### ISSUE TITLE
Usersnap Feedback - I reviewed three cases and saw this problem in all three. It looks like th[...]

### ISSUE THREAD

**Issue by paninee** [participedia/usersnaps#1446]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: I reviewed three cases and saw this problem in all three. It looks like the translation of the narrative heading for "Participant Recruitment and Selection" isn't working correctly.
[Open #1128 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/1128)

<a href='https://usersnappublic.s3.amazonaws.com/2021-09-07/22-52/f39e16bf-faa1-4ed8-88ac-914267f61707.png'>![Usersnap Feedback - I reviewed three cases and saw this problem in all three. It looks like th(...)](https://usersnappublic.s3.amazonaws.com/2021-09-07/22-52/0e4a9b90-f3df-4e81-805f-f814e7469b66.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2021-09-07/22-52/f39e16bf-faa1-4ed8-88ac-914267f61707.png'>Download original image</a>

**Browser**: Firefox 91 (Windows 10)
**Referer**: [https://participedia.net/case/6080](https://participedia.net/case/6080)
**Screen size**: 1760 x 1100 **Browser size**: 1760 x 939

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

--

A friendly reminder from Usersnap 👋

There’s a new widget in town. Make your bug reporting process sleeker and more advanced by changing to the new project template.

Learn more about how to transfer your project (and what new features you’ve been missing out on) in this [blog post](https://usersnap.com/blog/platform-new-widget/).

**Comment by jesicarson** [comment]
Hi @paninee, please have a look at this when you can. Likely to do with the recent update to chinese verified strings from phrase.

**Comment by paninee** [comment]
New entry get translated correctly now, but before we have the wrong translation from Phrase. The old entries may need manual data update as we found them. I updated https://participedia.net/case/6080 already.

![image](https://user-images.githubusercontent.com/29439995/152577768-75284b70-b21f-4306-a648-e184403bd841.png)
![image](https://user-images.githubusercontent.com/29439995/152577793-7a243a42-16e2-4d7e-b861-e7c35a37c1a8.png)
